superset-frontend/src/visualizations/TimeTable/TimeTable.jsx

```diff
@@ -89,7 +90,74 @@ const defaultProps = {
   url: '',
 };
 
+const TimeTableStyles = styled.div`
+  height: ${props => props.height}px;
+
+  th {
+    z-index: 1; // to cover sparkline
+  }
+`;
+
 class TimeTable extends React.PureComponent {
+  memoizedColumns = memoize(columnConfigs => [
```

Comment:
While this memoized function only takes one parameter, and thus is not subject to _that_ particular pitfall of lodash/memoize, it might be worth using `memoize-one` instead, or refactoring to a function component and using `useMemo` for safety.